packages/query-core/src/utils.ts

```diff
@@ -345,3 +346,8 @@ export function addToStart<T>(items: Array<T>, item: T, max = 0): Array<T> {
   const newItems = [item, ...items]
   return max && newItems.length > max ? newItems.slice(0, -1) : newItems
 }
+
+export const skipToken = Symbol.for(
+  'QUERY/skipToken-ea7c3132-6b5b-441e-89d8-4a98dcb5eb23',
+)
```

Comment:
added UUID to make extra sure we avoid conflicts.

Reply:
Shouldn't this just be `Symbol('QUERY/skipToken')` - then you don't have to worry about collisions at all. Since you have to use this variable to check for equality anyway.

Reply:
Im using symbol.for in here which might return already existing Symbol.

I prefer to use symbol.for because some other developers might want to not import the symbol itself, just get it from the "global registry of symbols". Very rare case but might happen.

![image](https://github.com/TanStack/query/assets/25470423/45e3fe96-b8ed-4bfa-94e7-945cbaae162c)